themes/theme-ios11/pages/Product/components/Options/components/TextOption/index.jsx

```diff
@@ -0,0 +1,160 @@
+import React, { Fragment, PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import Transition from 'react-transition-group/Transition';
+import TextField from '@shopgate/pwa-ui-shared/Form/TextField';
+import InfoIcon from '@shopgate/pwa-ui-shared/icons/InfoIcon';
+import withShowModal from '@shopgate/pwa-common/helpers/modal/withShowModal';
+import transition from './../../../Characteristics/Characteristic/transition';
+import { ProductContext } from '../../../../context';
+import OptionInfo from './components/OptionInfo';
+import styles from './style';
+
+/**
+ * @param {Object} props The compoent props.
+ * @returns {JSX}
+ */
+class TextOption extends PureComponent {
+  static propTypes = {
+    /** @type {Conditioner} */
```

Comment:
Why is that needed here? We don't create a Documentation from JSDoc, so a typeDef isn't needed.